packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTask.userKeystrokes.spec.js

```diff
@@ -180,6 +180,17 @@ describe('SurveyTask with user keystrokes', function () {
       expect(choiceImages).to.be.ok()
     })
 
+    it('should close choice on Escape key', async function () {
+      // tabbing to the first choice (Aardvark) and pressing Enter to open the choice (Aardvark)
+      await user.keyboard('[Enter]')
+      // tabbing into the choice component
```

Comment:
Shouldn’t the choice receive focus as soon as it opens? How would a blind volunteer use this, on an audio survey workflow?

Reply:
Added focus to the choice component, `aria-labelledby` for the focused component, and `aria-labelledby` for the choice question input groups. I tested with VoiceOver and navigating into and within the Choice seems improved to me, but please let me know what you think (no rush).

Reply:
Yes it should, great catch! I updated code to focus Choice component on open.